### PR TITLE
Set fall back currency EUR if language is DE

### LINKED ISSUE #1563: Fallback currency depending on user Location
From @sagararyal 
We need to fallback the default currency depending on the location of the user.
Example: if user in Germany, either language is German/English we need to set currency EUR.
If India we will set language English and currency as INR

@norbertschuler what do you think.

### PR DESCRIPTION
**Pull request by mahmed0715**
We will do vast implementation based on user location later. for now we set EUR if language is DE. 
@sagararyal please check this.

Fix #1563

### CODE REVIEW

File: app/actions/globalCurrency.js [hunk 1]
```diff
@@ -11,7 +12,11 @@ export function getPreferredCurrency() {
 }
 
 function guess() {
-  cache.currency = getItemSync('preferredCurrency') || context.currency;
```
Comment (norbertschuler) [inline app/actions/globalCurrency.js]: What about the native apps? https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/a42b62adfd32e14325203be1da702f07564e20fc/app/actions/globalCurrency.native.js#L13

Comment (sagararyal) [reply app/actions/globalCurrency.js]: this hasn't yet been implemented in native apps

Comment (norbertschuler) [reply app/actions/globalCurrency.js]: Ok, but I thought if we have the same code also already in the native app, we would have to adapt it there as well. But I haven't yet checked deeper, so @mahmed0715 can decide best if the linked code also should be adapted.

Comment (mahmed0715) [reply app/actions/globalCurrency.js]: I think its okay to merge this for now. Ultimate result is same for both. 
BTW I am planning a location based currency set mechanism now. That will be adapted both native and web end.

**Pull request merged**